oaknational/Oak-Web-Application
# [CUR-986] DocX front cover uses Cloudinary icons

## Description

Music year: {owa_music_year}

- Updated icon to be pulled from Cloudinary
- If not a valid subject icon, then `books` will be used

## How to test

1. Go to {owa_deployment_url}
2. Click on any subject/phase and download
3. You should see the most relevant icons

## Screenshots
How it should now look:
{screenshots}
<img width="558" alt="Screenshot 2024-10-23 at 11 53 14" src="https://github.com/user-attachments/assets/99154a42-c80b-4ef5-a115-2d3b83ac2597">
<img width="584" alt="Screenshot 2024-10-23 at 11 53 35" src="https://github.com/user-attachments/assets/1a09e08f-cff9-4b0d-a651-183f22758210">

## Checklist
